test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -9,6 +9,37 @@
 import org.junit.Test;
 
 public class UtilsTest {
+    
+    @Test
+    public void isAnyNull() throws Exception {
+        // only one object
+        assertFalse(Utils.isAnyNull(1));
+        assertFalse(Utils.isAnyNull(""));
+        assertFalse(Utils.isAnyNull(String.valueOf(1)));
+        assertFalse(Utils.isAnyNull("abc"));
+        
+        // null cast to Object
+        assertTrue(Utils.isAnyNull((Object) null));
+        
+        // all objects are not null
+        assertFalse(Utils.isAnyNull("abc", "ab", "a"));
+        assertFalse(Utils.isAnyNull(1, 2));
+        assertFalse(Utils.isAnyNull("abc", 3));
+        assertFalse(Utils.isAnyNull(1, Integer.valueOf(1)));
+        
+        // some objects are null
+        assertTrue(Utils.isAnyNull("abc", null, "a", null));
+        assertTrue(Utils.isAnyNull(1, 2, null));
+        assertTrue(Utils.isAnyNull("abc", null, 3));
+        assertTrue(Utils.isAnyNull(1, Integer.valueOf(1), null, null));
+        assertTrue(Utils.isAnyNull(String.valueOf(1), null));
+        
+        // all nulls
```

Comment:
Test cases usually should try to cover different cases and arguments. For many of your tests such as the four below, they are not significantly different. It is clear that if `Utils.isAnyNull(null, null, null)` then `Utils.isAnyNull(null, null, null, null)` will return too because it is a varadic function.